src/RequestLog/Controllers/RequestLogController.php

```diff
@@ -6,11 +6,18 @@
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\DB;
 use App\Http\Controllers\Controller;
-use Illuminate\Support\Facades\Cache;
 use Cego\RequestLog\Models\RequestLog;
+use Cego\RequestLog\Services\RequestLogOptionsService;
 
 class RequestLogController extends Controller
 {
+    protected $requestLogOptionsService;
+
+    public function __construct(RequestLogOptionsService $requestLogOptionsService)
```

Comment:
Tilføj den nede i de endpoints hvor du har brug for den istedet

Reply:
Laravel dokumentation siger god for det sådan her :) https://laravel.com/docs/9.x/controllers#dependency-injection-and-controllers